contrib/get-authors

```diff
@@ -22,8 +22,16 @@ while [ $# -gt 0 ]; do
   f=$1
   shift
   contributors=
+  has_header=`grep " \*\* Top contributors" $f`
+  if [ "x$has_header" == "x" ]
```

Comment:
(Hmm, somehow my suggestion didn't make it, posting it again)

Suggestion: The two lines above could just be written in a single line as `if grep -q " \*\* Top contributors" $f` (the then and the else branch would have to be swapped).